content/docs/02.installation/08.aws-ec2.md

```diff
@@ -69,7 +70,7 @@ Note that if you haven't set up basic authentication in the previous step, your
 ::
 
 
-![ec2 inbound rules](/docs/administrator-guide/deployment/aws-ec2/ec2_security_group_port_inbound_rules.png)
+![ec2 inbound rules](../../../public/docs/administrator-guide/deployment/aws-ec2/ec2_security_group_port_inbound_rules.png)
```

Comment:
```suggestion
![ec2 inbound rules](/docs/administrator-guide/deployment/aws-ec2/ec2_security_group_port_inbound_rules.png)
```

the same applies to all images, you don't need the ../../ — public is used as root for rendering images so you can reference all images starting from that path e.g. /docs/your_folder/your_image.png